hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -480,9 +482,10 @@ public void stop() {
 
   }
 
-  private void terminate(IOException ex) {
+  private void terminate(IOException ex, OMResponse omResponse) {
     String message = "During flush to DB encountered error in " +
-        "OMDoubleBuffer flush thread " + Thread.currentThread().getName();
+        "OMDoubleBuffer flush thread " + Thread.currentThread().getName() +
+        " when handling OMRequest: " + omResponse;
```

Comment:
What string does this print for the OMResponse? I'm not sure it will actually be the name of the response proto message.

Reply:
Updated in the description.
I got the output by just print the Response before handling it.